Models/testmodel1/config.cpp

```diff
@@ -0,0 +1,49 @@
+#include "config.hpp"
+
+//TODO: Consider where these should be initialized
+TempPWMDriver PWMDriver;
+TempDSHOTDriver DSHOTdriver;
+
+Motor_t motors[] = {
+    {   //Yaw servo motor
+        .axis = yaw,
+        .direction = positive,
+        .driver = PWMDriver,
+        .interfaceID = 0
+    },
+    {   //Roll BLDC motor
+        .axis = roll,
+        .direction = negative,
+        .driver = DSHOTdriver,
+        .interfaceID = 0
+    }
+};
+
+const uint8_t NUM_MOTORS = sizeof(motors)/sizeof(Motor_t);
+
+PID_t flightmode1Tuning = {
+    .p = 1.0f,
+    .i = 1.0f,
+    .d = 1.0f
+};
+
+ControlLimits_t flightmode1Limits = {
```

Comment:
I assumed `flightmode1Tuning` and `flightmode1Limits` would be in a struct together in a list of all flight modes, and also including a reference to the flight mode itself. (similar to how you have done for motors, just with a few nested structs)

Reply:
Does the flight mode organization look better now?

Reply:
I like it!

Reply:
I like it, but I'm curious if we could just use the same technique as the drivers use to that AM is the one that owns the instantiated driver, instead of making it static from the config file?
If not, I'm happy with this